app/views/handlers/list_action.py

```diff
@@ -18,7 +18,10 @@ def parent_location(self):
         )
 
     def _get_routing_path(self):
-        return self.router.routing_path(section_id=self.parent_location.section_id)
+        return self.router.routing_path(
+            section_id=self.parent_location.section_id,
+            list_item_id=self.parent_location.list_item_id,
```

Comment:
I think this might lead to confusion, as a parent wouldn't have a list item id?. So either don't set it, or set it to `None`. Otherwise, it makes me think that there is a scenario in which the parent will have a list item id. Worth adding doc strings also.

Reply:
Ya, a list collector in a repeating section shouldn't be allowed.

Reply:
removed list_item_id and added comment 👍